### ISSUE TITLE
Test Plugins in isolation

### ISSUE THREAD

**Issue by mattvb91** [corejam/corejam#9]
We need to test each package (Auth, DerShop) in isolation to make sure they work on their own.

So instead of running cypress agains the nextjs build we should run cypress against the individual stencil builds of the apps. This will also help speed up development branch pipelines.

- [x] Run Cypress against Auth Package build #11 
- [ ] Run Cypress against DerShop Package build
- [ ] Vercel deployment & test only runs as an optional succeed action on master or dev branch.

**Comment by mattvb91** [comment]
Blocked by #51

**Comment by mattvb91** [comment]
@patrickhaug I cant remember but I think we still have an outstanding problem with the router I think it was to be able to run all the tests locally?

I think Vercel may be turning into a blocker for dependabot as everything is timing out to wait for other builds. Need a pro account for parallel deployments

**Comment by patrickhaug** [comment]
Yup. Router handling is suboptimal, we should provide some base components that help up with prerender and guarded auth state.
